hr_loan_management/README.rst

```diff
@@ -0,0 +1,180 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+========================
+Employee Loan Management
+========================
+
+This module add functionality to:
+
+1. Manage employee loan request
+2. Import employee loan realization into bank statement
+3. Track employee loan repayment
+
+Installation
+============
+
+To install this module, you need to:
+
+1.  Clone the branch 8.0 of the repository https://github.com/OCA/hr
+2.  Add the path to this repository in your configuration (addons-path)
+3.  Update the module list
+4.  Go to menu *Setting -> Modules -> Local Modules*
+5.  Search For *Employee Loan Management*
+6.  Install the module
+
+Configuration
+=============
+
+**Employee Loan Type**
+
+To configure employee loan type, you need to:
+
+1. Go to menu *Human Resource -> Configuration -> Employee Loan -> Employee Loan Type*
+2. Create new employee loan type
+
+
+Usage
+=====
+
+Create Employee Loan Request
+----------------------------
+
+1. Go to menu *Human Resources -> Employee Loan -> My Loan*
+2. Create employee loan
+3. Click *Confirm* button to confirm employee loan request
+
+Approve Employee Loan Request
+-----------------------------
+
+1. Go to menu *Human Resources -> Employee Loan -> My Loan*
+2. Open employee loan data with *Waiting for Approval* status
+3. Click *Approve* button
+
+Odoo will create journal entry with bellow configuration:
+
+Loan Principle Receivable Dr.
+    Loan Realization Cr.
+
+Note:
+*Loan principle receivable* will be generated as much as number of payment schedule
+
+Employee Loan Realization
+-------------------------
+
+Employee loan can be realized by reconcile *Loan Realization Move Line*
+created from step above. You can reconcile them by using any accounting
+method provided by Odoo, such as: (1) bank statement, (2) voucher, or
+(3) manualy creating journal entry and reconciliation.
+
+
+Employee Loan Principle Repayment
+---------------------------------
+
+Employee loan's principle can be paid by reconcile *Principle Receivable Move Line*
+created from approval steap above. You can reconcile them by using any accounting
+method provided by Odoo, such as: (1) bank statement, (2) voucher, or
+(3) manualy creating journal entry and reconciliation.
```

Comment:
There is a use case, where the employer allows for the employee to repay a large amount in several settlements.
For example, to repay $1000 in 10 x $100 payments.
This is usually done through the Payroll Payslip.
Can this scenario be easily covered? It would be nice to describe how in the Readme.